param_templates/MOM_input.yaml

```diff
@@ -526,10 +526,10 @@ Global:
         datatype: logical
         units: Boolean
         value:
-            $OCN_GRID == "gx1v6": True
-            $OCN_GRID == "tx0.66v1": True
-            $OCN_GRID == "tx2_3v2": True
-            $OCN_GRID == "tx0.25v1": True
+            $RUN_TYPE != "hybrid" or $CONTINUE_RUN == True and $OCN_GRID in ["gx1v6", "tx0.66v1", "tx2_3v2", "tx0.25v1"]:
+                True
+            else:
+                False
```

Comment:
Can you add parenthesis to these? `($RUN_TYPE != "hybrid" or $CONTINUE_RUN == True) and $OCN_GRID in ["gx1v6", "tx0.66v1", "tx2_3v2", "tx0.25v1"]` and `$RUN_TYPE != "hybrid" or ($CONTINUE_RUN == True and $OCN_GRID in ["gx1v6", "tx0.66v1", "tx2_3v2", "tx0.25v1"])` are different statements, and I'm not clear on what the order of operations are. If I had to guess, since `and` is multiply and `or` is addition, the second one is what python is actually doing?

Just checked, and I guessed right:

```py3
>>> True or True and False
True
>>> True or (True and False)
True
>>> (True or True) and False
False
```

So if `CONTINUE_RUN` is True and the grid is one of the global grids we support, or if `RUN_TYPE` is not hybrid, then `INIT_LAYERS_FROM_Z_FILE` is True. Otherwise (`RUN_TYPE` is hybrid and either `CONTINUE_RUN` is False or we aren't on a supported grid) it is False. Is that what we want?

Reply:
Reading through MOM_state_initialization module, it actually looks like the check for `CONTINUE_RUN` is unnecessary, so I'll remove it once I confirm that is indeed the case. Hence, there should be no need for parantheses, but thanks for catching this.